deno/send_message/src/functions/send_message_discord_webhook.ts

```diff
@@ -0,0 +1,23 @@
+export default async function (env: any, message: string) {
+  const webhook = env["DISCORD_WEBHOOK_URL"];
+  console.log(webhook);
+
+  const res = await fetch(webhook, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+      content: message,
+    }),
+  });
```

Comment:
It looks like this can throw an error in cases where the webhook URL was not supplied and then the function returns a very cryptic error response:

```shell
$ curl http://localhost:3000/ -d '{"variables": {},"payload": {"type": "Discord","message": "Programming is fun!"}}' -H "X-Internal-Challenge: secret-key"
{"stdout":"","stderr":"Code file not found."}
```

It would be really helpful to improve this.

Reply:
This error is not code error

It is actually a system error..

I will check this once more...

Reply:
Ya, it's an Open Runtimes error that occurs because of an error in the function code. It would be nice if the function code caught errors and supplied a helpful message.